src/components/Sortable.vue

```diff
@@ -84,7 +86,8 @@ watch(containerRef, (newDraggable) => {
       onSort: (event) => emit("sort", event),
       onRemove: (event) => emit("remove", event),
       onFilter: (event) => emit("filter", event),
-      onMove: (event, originalEvent) => emit("move", event, originalEvent),
+      // See https://github.com/MaxLeiter/sortablejs-vue3/pull/56 for context on `attrs`.
+      onMove: (event, originalEvent) => "onMoveCapture" in attrs ? (<() => void>attrs.onMoveCapture)() : emit("move", event, originalEvent),
```

Comment:
Should we pass `event` and `originalEvent` to `onMoveCapture`?

Reply:
Oh, sorry. I skipped this moment. Yes, we must pass args.

Reply:
I try to update the Events section, but my English is bad.

Reply:
No worries, just 1 line is needed here showing how to use @move.capture:  
<img width="864" alt="image" src="https://user-images.githubusercontent.com/8675906/209722375-a6bc41a4-1581-4ddb-a87e-bc4d0803ae0d.png">